tests/Api/Admin/ChannelsTest.php

```diff
@@ -44,6 +44,29 @@ public function it_gets_a_channel(): void
         );
     }
 
+    /** @test */
+    public function it_deletes_a_channel(): void
+    {
+        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'channel.yaml']);
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        $this->client->request(
+            method: 'DELETE',
+            uri: '/api/v2/admin/channels/MOBILE',
+            server: $header,
+        );
+
+        $this->assertResponseCode($this->client->getResponse(), Response::HTTP_NO_CONTENT);
+
+        $this->client->request(
+            method: 'GET',
+            uri: '/api/v2/admin/channels/MOBILE',
+            server: $header,
+        );
+
+        $this->assertResponseCode($this->client->getResponse(), Response::HTTP_NOT_FOUND);
```

Comment:
You can add a similar call before deletion to prove the channel is existing before.